specification/recoveryservicesdatareplication/resource-manager/Microsoft.DataReplication/preview/2021-02-16-preview/recoveryservicesdatareplication.json

```diff
@@ -0,0 +1,7002 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Site Recovery Management Service API",
+    "description": "A first party Azure service enabling the data replication.",
+    "version": "2021-02-16-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DataReplication/replicationFabrics/{fabricName}/fabricAgents/{draName}": {
+      "get": {
+        "tags": [
+          "Dra"
```

Comment:
Would you please clarify that it's tagged "Dra" but it's called "fabric agents" in the api path?

Reply:
If you don't fix your models now/soon, it will be a breaking change for later. And anyone who consumed the proposed name will have adjust. Consider you might actually be saving work in the long run to fix it as soon as you can.

Reply:
I will need to discuss this change not only my immediate team but also teams that share the same code space but not going public for model change to happen. Since this is going to be published for Public Preview that is happening in a month, will it be ok to introduce breaking change later for GA and for now we will create a work item to track this progress?

Reply:
I agreed with Tim, at least the Models should be fixed, because it will cause sdk breaking change too.

Reply:
We have created a [Task Item](https://msazure.visualstudio.com/One/_workitems/edit/24973958) to track this change. Is it possible to obtain some sort of exception for this release as it's for public preview on 9/30/23 and this change would require significant amount of efforts and time. If possible, we would like to resolve it for GA. Additionally, majority of the clients of our service is expected to use Azure portal or PowerShell that we are developing to interact with us so we shouldn't need to worry about SDKs too much.

Reply:
As long as you commit to implement this by GA we are good.